survey/survey_realtime_streaming.py

```diff
@@ -249,6 +182,53 @@ def flatten(x, name=''):
     flatten(y)
     return out
 
+# # Define function to check if survey submission Id exists in Druid
+def check_survey_submission_id_existance(key,column_name,table_name):
+    try:
+        # Establish connection to Druid
+        url = config.get("DRUID","sql_url")
+        url = str(url)
+        parsed_url = urlparse(url)
+
+        host = parsed_url.hostname
+        port = int(parsed_url.port)
+        path = parsed_url.path
+        scheme = parsed_url.scheme
+
+        conn = connect(host=host, port=port, path=path, scheme=scheme)
+        cur = conn.cursor()
+        # Query to check existence of survey submission Id in Druid table
+        query = f"SELECT COUNT(*) FROM \"{table_name}\" WHERE \"{column_name}\" = '{key}'"
+        cur.execute(query)
+        result = cur.fetchone()
+        count = result[0]
+        infoLogger.info(f"Found {count} entires in {table_name}")
+        if count == 0:
+            return False
+        else:
+            return True        
+    except Exception as e:
+        infoLogger.info(f"Failed to check the {table_name} datasource")
+        # Log any errors that occur during Druid query execution
+        errorLogger.error(f"Failed to check the {table_name} datasource {e}",exc_info=True)
+
+def set_null_value(data):
+    if "userProfile" in data :
+        if config.get("OUTPUT_DIR","CAPTURE_USER_PROFILE") == "True":
```

Comment:
Why are we passing null if CAPTURE_USER_PROFILE and CAPTURE_ORGANISATION_NAME is true. It should be reversed